tests/webhook_test.go

```diff
@@ -11,7 +11,10 @@ func (c *ClientTests) TestWebhookCreate() {
 	client := c.TestClient()
 	assert, require := c.Assert(), c.Require()
 
-	webhook, err := client.CreateWebhook(c.fixture.WebhookUrl())
+	webhook, err := client.CreateWebhookWithDetails(
+		&easypost.CreateUpdateWebhookOptions{
+			URL: c.fixture.WebhookUrl(),
+		})
```

Comment:
Issue: please correct these in a separate PR as they have nothing to do with billing.

Reply:
Resolved